### DISCUSSION TITLE
Disable output of _version field for WATCH?

### DISCUSSION THREAD

**Discussion by daledude** [ClickHouse/ClickHouse#35063] [Q&A] [answered]
I'm trying to use clickhouse-client to use WATCH to simply "tail -f" a table but it's outputting the _version field which I don't want. Possible to disable that instead of having to pipe to something like jq which ends up messing the output by constantly shifting the output to the right.

**Comment by alexey-milovidov** [comment, marked as answer]
LIVE VIEW is mostly unsupported, this feature is still experimental and can be changed in backward incompatible way at any time.
